client/src/components/PosterList.js

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+import Poster from './Poster'
+
+PosterList.propTypes = {
+  list: PropTypes.array,
+}
+
+export default function PosterList({ list }) {
+  return (
+    <StyledPosterList>
+      {list.map(({ id, poster_path, name }) => {
+        return (
+          <li key={id}>
+            <Poster
+              path={
+                poster_path && `https://image.tmdb.org/t/p/w300/${poster_path}`
+              }
+              alt={`Poster von ${name}`}
+            />
+          </li>
+        )
+      })}
+    </StyledPosterList>
+  )
+}
+
+const StyledPosterList = styled.ul`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 16px;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+
+  li {
+    display: flex;
+    border-radius: 4px;
+
+    img {
```

Comment:
I would style the image directly in your Poster component. Now you have styles here and there, it'd would add for a better overview, if you have them in one place.

Reply:
Or do you want to override the styles from your Poster component?

Reply:
Yes, these are just styles for the Poster inside PosterList. Therefore i didn't add them to the Poster component.